### PR TITLE
Create 0032-cross-sectoral-standards.md

### PR DESCRIPTION
**Pull request by bhadley**
Creation of draft ADR for cross sectoral standards attribute extensibility. Next step is to build out the two proposals as tech spec options such that we have options to review with the initiatives and community to enable consensus.

### CODE REVIEW

**Review by raimundo-henriques — APPROVED** [review]

**Comment by raimundo-henriques** [comment]
Thank you @bhadley for the fixes!

File: decisions/log/0032-cross-sectoral-standards.md [hunk 1]
```diff
@@ -61,8 +62,6 @@ We have also considered the possibility of disclosing a cross sectoral standard
         :: for version 2.1 of the PACT Framework
         : GHGP Product
         :: for the Greehouse Gas Protocol (GHGP) Product Standard
-        : PAS 2050
-        :: for the Publicly Available Specification (PAS) 2050
         : ISO14040-44
         :: for the ISO 14044-44 Standard
```
Comment (JohnVonNeumann) [inline decisions/log/0032-cross-sectoral-standards.md:66]: Key states "ISO14040-44" and value states "ISO 14044-44"

Comment (raimundo-henriques) [reply decisions/log/0032-cross-sectoral-standards.md:66]: Great catch! Thank you! Will be fixed asap!

**Review by Akiko-JEITA — APPROVED** [review]
Approval. 
For your reference, Green x Digital Product framework, GHG Protocol scope3 standard, Green x Digital scope3 framework, and ISO14025, 14027 will be included options in "the Green x Digital Consortium's Tech Specs Ver.2" In addition, other standards not contained the option will be able to be described.

File: decisions/log/0032-cross-sectoral-standards.md (hunk 1 above)
Comment (bhadley) [reply decisions/log/0032-cross-sectoral-standards.md:66]: Thanks for the catch! Now fixed (although note we are still awaiting final alignment from methodology colleagues on this list)

**Comment by bhadley** [comment]
Thanks @Akiko-JEITA  for the feedback (summarized below for reference):
> Approval. For your reference, Green x Digital Product framework, GHG Protocol scope3 standard, Green x Digital scope3 framework, and ISO14025, 14027 will be included options in "the Green x Digital Consortium's Tech Specs Ver.2" In addition, other standards not contained the option will be able to be described.

I synced with Joy to discuss your feedback. We have the following clarifications:
1. We will include standards in this list which specify a cross-sector methodology for the calculation of a PCF. Because GHGP Scope 3 standard is at the corporate level, we will not include it in the set. Because Green x Digital Product framework is aligned to PACT Framework, we do not see the need to include this in addition to the PACT Framework. Organizations specifically using GxD references can specify this in the comments.
2. Our understanding of ISO14025 and ISO14027 is that these standards specify principles and procedures for EPDs and PCRs within EPDs, and therefore don't specify PCF methodology beyond ISO14040-44 and ISO14067. Is this your understanding as well, or do these standards bring additional methodological clarifications?

**Comment by Akiko-JEITA** [comment]
> Thanks @Akiko-JEITA for the feedback! We accidentally closed out this PR but will be opening a new one to finalize the discussion and consensus on the topic.
> 
> Regarding your feedback (summarized below for reference)
> 
> > Approval. For your reference, Green x Digital Product framework, GHG Protocol scope3 standard, Green x Digital scope3 framework, and ISO14025, 14027 will be included options in "the Green x Digital Consortium's Tech Specs Ver.2" In addition, other standards not contained the option will be able to be described.
> 
> I synced with Joy to discuss your feedback. We have the following clarifications:
> 
> 1. We will include standards in this list which specify a cross-sector methodology for the calculation of a PCF. Because GHGP Scope 3 standard is at the corporate level, we will not include it in the set. Because Green x Digital Product framework is aligned to PACT Framework, we do not see the need to include this in addition to the PACT Framework. Organizations specifically using GxD references can specify this in the comments.
> 2. Our understanding of ISO14025 and ISO14027 is that these standards specify principles and procedures for EPDs and PCRs within EPDs, and therefore don't specify PCF methodology beyond ISO14040-44 and ISO14067. Is this your understanding as well, or do these standards bring additional methodological clarifications?

 Thank you @bhadley for your feedback. I would like to clarify our comments as follows.

- The Green x Digital CO2 Visualization Framework consists of “Green x Digital Product Framework” as product-based calculation method and “Green x Digital Scope 3 Framework” as organization-based calculation method. To reflect this, the Green x Digital’s Tech Specs Ver 2 will include GHG Protocol scope3 standard as well as GxD Product Framework and GxD Scope 3 Framework in the option as the Green x Digital unique specification.

- The Green x Digital CO2 Visualization Framework Ver 2 will allow the CO2 data based on the Japan EPD Program “the SuMPO EDP” as one of the data can be shared as well as PCF conforming to other cross sectoral standards. Also, we are considering whether “the SuMPO EDP” would be regarded as one of the Product-specific rules in the Green x Digital CO2 Visualization Framework. As the SuMPO EDP is based on ISO14025 and 14027, the Green x Digital’s Tech Specs Ver 2 will include those standards in the option as the Green x Digital unique specification.

**Comment by bhadley** [comment]
Thanks @Akiko-JEITA  so much for the clarifications! That fully clarifies your feedback, thank you. We will proceed then with the proposal per your endorsement, and in parallel as discussed are happy to review the Green x Digital specification.

Note that we accidentally closed this ADR before it has been reviewed by the broader PACT community, so we have opened a new PR linking to this ADR. All future comments regarding this proposal should be included in the new ADR: https://github.com/wbcsd/data-exchange-protocol/pull/80

**Pull request merged**